mods/cnc/maps/funpark01/scj01ea.lua

```diff
@@ -0,0 +1,52 @@
+RifleReinforcments = { "e1", "e1", "e1", "bike" }
+BazookaReinforcments = { "e3", "e3", "e3", "bike" }
+BikeReinforcments = { "bike" }
+
+
+ReinforceFromSea = function(passengers, entry, exit)
+	Reinforcements.ReinforceWithTransport(nod, "oldlst", passengers, { entry, exit  }, { entry })
+	Media.PlaySpeechNotification(nod, "Reinforce")
+end
+
+WorldLoaded = function()
+	nod = Player.GetPlayer("Nod")
+	dinosaur = Player.GetPlayer("Dinosaur")
+	
+	NodObjective1 = nod.AddPrimaryObjective("Investigate the nearby village for reports of strange activity")
```

Comment:
You will need a new line somewhere:
![dinodescription](https://cloud.githubusercontent.com/assets/7704140/5612593/e4b1756a-94d7-11e4-89b4-4b0ae89e9ba7.png)